pavelib/utils/test/bokchoy_utils.py

```diff
@@ -44,11 +44,12 @@ def start_server(cmd, logfile, cwd=None):
         cmd = (
             "DEFAULT_STORE={default_store} "
             "coverage run --rcfile={coveragerc} -m "
-            "manage {service} --settings bok_choy runserver "
+            "manage {service} --settings {settings} runserver "
             "{address} --traceback --noreload".format(
                 default_store=options.default_store,
                 coveragerc=coveragerc,
                 service=service,
+                settings=Env.SETTINGS,
```

Comment:
Not for this PR or JIRA ticket, but does it make sense to have other paver commands auto-detect the proper devstack default settings?